packages/itwinui-react/src/core/SideNavigation/SideNavigation.tsx

```diff
@@ -150,16 +151,7 @@ export const SideNavigation = React.forwardRef((props, forwardedRef) => {
         </Box>
         {expanderPlacement === 'bottom' && ExpandButton}
       </Box>
-      {submenu && (
-        <WithCSSTransition
-          in={isSubmenuOpen}
-          dimension='width'
-          timeout={200}
-          classNames='iui'
-        >
-          {submenu}
-        </WithCSSTransition>
-      )}
+      {submenu ?? submenu}
```

Comment:
This can be simplified to just `submenu`. The `a ?? b` notation means "if `a` is undefined, call `b`", so it's essentially calling `submenu` whether its `undefined` or not

I noticed there is a unit test that is failing and changing this to `isSubmenuOpen && submenu` seemed to fix this test.